src/viam/sdk/components/board.cpp

```diff
@@ -23,7 +23,14 @@ API API::traits<Board>::api() {
 Board::status Board::from_proto(const viam::component::board::v1::Status& proto) {
     Board::status status;
     for (const auto& analog : proto.analogs()) {
-        status.analog_reader_values.emplace(analog.first, analog.second);
+		Board::analog_value result;
+		result.value = analog.second;
+		// The status does not contain the extra data to describe the accuracy of the reader. We
+		// fill those in with 0's instead.
+		result.min_range = 0.0;
+		result.max_range = 0.0;
+		result.step_size = 0.0;
```

Comment:
Discussion question: in the [protobufs](https://github.com/viamrobotics/api/blob/main/proto/viam/component/board/v1/board.proto), this extra data is present in `ReadAnalogReaderResponse` but not in `Status`. Perhaps the `Board::status` here should change from an `unordered_map<string, analog_value>` to an `unordered_map<string, int32_t>` and just continue to hold the raw values without the accuracy information. I'm unsure.

(Part of me wonders if this entire function was supposed to be removed in #231, when board components started using the same status stuff as all other components. I'm unsure if this is dead code...)

Reply:
...I think the way to actually resolve this tension is that the data stored in the status should not change: although we now get extra data in responses from reading analogs, we do _not_ get any such data in status messages, so should keep the old version of that.